agent/bench-scripts/unittests

```diff
@@ -27,14 +27,12 @@ export _testdir=${_testroot}/pbench-agent
 # Copy bench-scripts execution environment to _testroot
 export _testopt=${_testroot}/opt/pbench-agent
 res=0
-mkdir -p ${_testopt}/bench-scripts/postprocess ${_testopt}/bench-scripts/driver ${_testopt}/bench-scripts/templates ${_testopt}/util-scripts ${_testopt}/lib ${_testopt}/unittest-scripts ${_testopt}/common/lib ${_testopt}/config/benchmark
+mkdir -p ${_testopt}/bench-scripts/postprocess ${_testopt}/bench-scripts/driver ${_testopt}/bench-scripts/templates ${_testopt}/util-scripts ${_testopt}/lib ${_testopt}/unittest-scripts ${_testopt}/common/lib ${_testopt}/config
```

Comment:
We also create `${_testopt}/config` at line 219...do we care?

Reply:
It is still needed, see the uses of `${_testopt}/config` throughout the `unittests` script.